docs/UserGuide.md

```diff
@@ -340,6 +357,18 @@ Examples:
 * `C add n/John Doe p/98765432 e/jon@jon.com t/CS2020`
 * `C add n/Rajesh Kumar p/98762342 e/raj@law.com t/Friend`
 
+:information_source: The `t/TAG` parameter is optional. You can use it as many times as you want (i.e. `C add n/John
+ Doe p/98765432 e/jon@jon.com t/CS2020 t/Brother` will add 2 tags to the contact instead of just 1).
+ 
+:information_source: To add a contact to a module, simply enter the module code as a tag. If the provided module code
+ does not exist, the tag will just be treated as another ordinary tag. No error will be thrown. 
+
+:information_source: You can add a contact using an email that already exists in TrackIt@NUS. It is fine for 2
+ contacts to have the same email.
+ 
+:information_source: Phone numbers can be of any length and begin with any digit. The only restrictions placed on phone
+ numbers is that they must all numbers.
```

Comment:
I think we can edit this sentence to this: " Phone numbers have to consist of just digits only, and can be of any length!"